Civi/Api4/SepaTransactionGroup.php

```diff
@@ -17,4 +17,13 @@ public static function get($checkPermissions = TRUE) {
       ->setCheckPermissions($checkPermissions);
   }
 
+  public static function permissions(): array {
+    return [
+      'get' => ['view sepa groups'],
+      'create' => ['batch sepa groups'],
+      'update' => ['batch sepa groups'],
+      'delete' => ['delete sepa groups'],
```

Comment:
I guess it is intended that it is not `['batch sepa groups', 'delete sepa groups']` like in `SepaSddFile` and `SepaContrinbutionGroup`. I cannot judge it right now, though I just comment to make sure it's correct :slightly_smiling_face: